hardware/laser_diode_drivers.py

```diff
@@ -18,7 +18,26 @@
 """
 
 import visa
+from hardware import u
 
+class MockLaserDiodeDriver:
+    def __init__(self, instr_name = None):
+        if not instr_name:
+            self.name = "Laser Diode Driver - ILX Lightwave 3724B"
+        else:
+            self.name = instr_name
+        self._current = 2 * u.amp

```

Comment:
Let's set the current to be a few tens of milliamps. Amps are actually quite large and tend to burn stuff.